docs/interactivity.qmd

```diff
@@ -107,7 +107,7 @@ You can also use the [Layout](#sec-layout) primitives (columns, panels, and tabl
 
 ## Formatting
 
-The `console.print()` method supports [formatting]((https://rich.readthedocs.io/en/stable/console.html)) using simple markup. For example:
+The `console.print()` method supports [formatting](https://rich.readthedocs.io/en/stable/console.html) using simple markup. For example:
```

Comment:
This one took me to https://inspect.aisi.org.uk/(https://rich.readthedocs.io/en/stable/console.html) which gave me a GitHub Pages 404.